src/app/board/active-quests/active-quests.component.html

```diff
@@ -0,0 +1,14 @@
+
+<h3 *ngIf="activeQuests.length !== 0" class="quests__title">
```

Comment:
You can make your HTML prettier if you use `<ng-template></ng-template>`

For example this way: 
```
<h3 *ngIf="activeQuests.length !== 0; else noTasksBlock" class="quests__title">
  MY ACCEPTED TASKS
</h3>
<app-quest ...></app-quest>
<ng-template #/noTasksBlock>
  <h3>
    No active tasks for today
  </h3>
</ng-template>
```